src/app/Fake.DotNet.Testing.Expecto/Expecto.fs

```diff
@@ -0,0 +1,135 @@
+/// Contains tasks to run [Expecto](https://github.com/haf/expecto) unit tests.
+module Fake.DotNet.Testing.Expecto
+
+open Fake.Core
+open Fake.Core.Environment
+open Fake.Core.StringBuilder
+open Fake.Testing.Common
+open System
+
+/// CLI parameters available if you use Tests.runTestsInAssembly defaultConfig argv in your code:
+type ExpectoParams =
```

Comment:
As it is part of the `Expecto`-module we could 
- either call it `Params` (if we want to, users would then use `Expecto.Params`)
- move it out of the `Expecto` module and leave it `ExpectoParams`

What do you think? We probably even should have some guideline around this and make it more uniform in fake 6

Reply:
Shortening to Params.